lib/ansible/galaxy/collection/__init__.py

```diff
@@ -544,8 +544,8 @@ def download_collections(
         for fqcn, concrete_coll_pin in dep_map.copy().items():  # FIXME: move into the provider
             if concrete_coll_pin.is_virtual:
                 display.display(
-                    'Virtual collection {coll!s} is not downloadable'.
-                    format(coll=to_text(concrete_coll_pin)),
+                    '{coll_type!s} collection {coll!s} is not downloadable'.
```

Comment:
`{coll!s}` will already render as `"{self.type} collection Git repo"`, resulting the above variant to look like `{coll_type!s} collection "{coll_type} collection Git repo" is not downloadable`. For example `git collection "git collection Git repo" is not downloadable`. This variant is better:
```suggestion
                    '{coll!s} is not downloadable'.
```
See https://github.com/ansible/ansible/blob/65a96daa/lib/ansible/galaxy/dependency_resolution/dataclasses.py#L464-L473.